packages/testlab/package.json

```diff
@@ -4,10 +4,12 @@
   "description": "A collection of test utilities we use to write LoopBack tests.",
   "main": "index",
   "scripts": {
-    "build": "tsc && npm run build:es2015",
-    "build:es2015": "tsc --target es2015 --outDir lib-es2015",
+    "build": "npm run build:es6 && npm run build:es8",
+    "build:es6": "tsc --target es2015 --outDir lib6",
+    "build:es8": "tsc --target es2017 --outDir lib",
```

Comment:
I was going with wikipedia versions -- https://en.wikipedia.org/wiki/ECMAScript (ES8 really isn't a thing and there are too many aliases. I'm going to go with build:lib + lib6 as you suggested for clarity.